apps/docs/src/app/(main)/api/layout.tsx

```diff
@@ -0,0 +1,16 @@
+export const metadata = {
+  title: 'Next.js',
+  description: 'Generated by Next.js',
+}
```

Comment:
Not really, and this has to do with the general redirecting of pages with the proxy. 
I just added this here since the app needs a "/" route, but in production that is rerouted to the framer page.

I think I'll nonetheless change these to some more real values just in case 👍